ass_man/models.py

```diff
@@ -20,62 +21,84 @@ def __str__(self):
         return (self.vendor + ' ' + self.model_number) or ''
 
 
-class Instance(models.Model):
+class Asset(models.Model):
     model = models.ForeignKey(Model, on_delete=models.PROTECT)
-    hostname = models.CharField(max_length=64)
+    hostname = models.CharField(max_length=64, blank=True, null=True)
+    datacenter = models.ForeignKey('Datacenter', on_delete=models.CASCADE, null=True)
```

Comment:
Should this be a cascade? I think we should be careful with deletion behavior

Reply:
I think we want this to be a protect- can't delete a data center if there exists an asset referencing it

Reply:
ok .. i was thinking if the whole datacenter like burned down or something you'd just wanna delete everything